### ISSUE TITLE
Gradle error

### ISSUE THREAD

**Issue by eduardorcosta** [tidev/titanium#592]
hello, I'm trying to build an application that was build with ti sdk 9. I don't have any erros with the application, but with the latest version of titanium it shows:
![Screen Shot 2022-08-09 at 14 03 17](https://user-images.githubusercontent.com/7453858/183712984-b4254fb9-c583-4461-8b30-0aff2fc7e525.png)

I'm using a mac studio (arm) and stucked in this error.

**Comment by m1ga** [comment]
Can you try this one: https://stackoverflow.com/a/69442569/5193915

**Comment by eduardorcosta** [comment]
Sorry, I don't have any Antivirus. I'm using macos.

**Comment by m1ga** [comment]
I'm just searching for the gradle error. This is another hit: https://stackoverflow.com/a/68668663/5193915
Not seen that error in Titanium before.

We had other users that uses Antivir on Mac that blocked some access or the internal Gatekeeper blocking files

**Comment by eduardorcosta** [comment]
Yes I did this, but even removing directory reinstalling things, everything. I canno't run, then I opened the issue.

**Comment by eduardorcosta** [comment]
ti build -p android

**Comment by m1ga** [comment]
it looks like you are running out of VSCode. Can you run the command from a normal terminal outside of VSCode please.

**Comment by eduardorcosta** [comment]
Shure!

**Comment by eduardorcosta** [comment]
I stopped every gradle process and remove the local directory, so the build process can download it again and install everything.
I started from a normal terminal this time, and the error continue
![Screen Shot 2022-08-23 at 11 32 45](https://user-images.githubusercontent.com/7453858/186185993-0fff4c8d-d96f-4082-bf63-ac9da88a73d8.png)
.

**Comment by m1ga** [comment]
did you build the ti 9 version before on the same Mac? Or was it an older Intel one?
Which Java are you using? Maybe its arm related, I'm using an Intel machine and didn't not see that error before

**Comment by eduardorcosta** [comment]
the project is on a NAS server, I was building this from version 8.
I'm using openjdk 11.0.16 2022-07-19
In my other mac with intel processor I have the same issue.

**Comment by m1ga** [comment]
ok, multiple things to try:
* don't use a NAS, try to build it locally 
* openjdk should be fine for a non-m1 build. `You can use brew install openjdk@11 or if you use a M1 Mac you can use the version from https://github.com/microsoft/openjdk-aarch64/releases/tag/jdk-16.0.2-ga` (from reports of other M1 users)
https://fromzerotoapp.com/how-to-install-appcelerator-titanium/#link9

**Comment by cb1kenobi** [comment]
Closing due to lack of activity.
